lib/messages.rb

```diff
@@ -27,7 +27,7 @@ def validation_message(errors)
     puts errors
   end
 
-  def pins_message(feedback)
+  def score_message(feedback)
     puts "Red pins: #{feedback[:red]} \nWhite pins: #{feedback[:white]}"
   end
```

Comment:
As mentioned in another comment, these methods should be defined as module level methods, which has cascading effects in the codebase. The include statements can be eliminated and the clarity of the code skyrockets.